torch/csrc/jit/symbolic_script.cpp

```diff
@@ -708,16 +715,32 @@ const std::vector<std::string> functions = {
                 output = bn_out
 
             def backward(grad_output):
-                if weight is not None:
-                    grad_output = grad_output * torch.t(weight)
-                    weight = grad_output * torch.t(bn_out)
+                if weight is not None and bias is not None:
+                    grad_bn_out = grad_output * weight
+                    grad_weight = grad_output * bn_out
```

Comment:
you do need sum_to_size here and for other grad_weight terms (sum_to_size is only unnecessary for grad_bn_out). Also, sum_to_size is needed for grad_bias terms. Why aren't tests catching this?

Reply:
hmmm yeah you are right, thanks for pointing out. I think the reason that the tests did not catch those is because we only check `grad_input` in our testing infra, I will see if I can change that.